lib/ncr/approval_manager.rb

```diff
@@ -0,0 +1,90 @@
+module Ncr
+  class ApprovalManager
+    attr_reader :work_order
```

Comment:
are we using this `attr_reader` outside of this class? If not, about about making it private?